build_rhel.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Pass any cmake options this way:
+
+# ./build_rhel.sh -DFLEDGE_INSTALL=/some_path/Fledge
+
+source scl_source enable devtoolset-7
+export CC=$(scl enable devtoolset-7 "command -v gcc")
+export CXX=$(scl enable devtoolset-7 "command -v g++")
+
+mkdir build
```

Comment:
If build directory already exists then this statement gives error `mkdir: cannot create directory ‘build’: File exists`, so we can add `-p` flag to mkdir command.